### ISSUE TITLE
KAFKA_TOOLS_LOG4J_LOGLEVEL=DEBUG not effective

### ISSUE THREAD

**Issue by cotedm** [confluentinc/cp-docker-images#339]
``KAFKA_TOOLS_LOG4J_LOGLEVEL=DEBUG`` is not effective for the console consumer tool in my attempts. Here is an example of the command I use that does not change the log level properly:

``docker run --rm -ti --net=host -e KAFKA_TOOLS_LOG4J_LOGLEVEL=DEBUG confluentinc/cp-kafka kafka-console-consumer --bootstrap-server localhost:29092 --topic performanceTest0 --new-consumer --from-beginning``

If I exec into the docker container while this is running, I can see clearly that `/etc/kafka/tools-log4j.properties` is still set to WARN. However, if I run the command at the end of the `configure` script, I can see that `/etc/kafka/tools-log4j.properties` gets updated properly:
https://github.com/confluentinc/cp-docker-images/blob/3.3.x/debian/kafka/include/etc/confluent/docker/configure#L123

So I suspect the problem is that `configure` is not called when running the console consumer.

**Comment by ewencp** [comment]
@cotedm That's true, it does not get executed. When you run a container and specify the command to run, it does not execute the `RUN` command included in the Dockerfile, e.g. https://github.com/confluentinc/cp-docker-images/blob/3.3.x/debian/enterprise-kafka/Dockerfile#L47. That's the script that does all the `configure` stuff before launching the actual target command.

I don't think docker provides much of a way around this. To make this work we'd probably have to make the `run` script support doing everything it normally does but then swapping out the final command executed and document that you need to prefix your commands with `/etc/confluent/docker/run`.

**Comment by cotedm** [comment]
@ewencp thanks for the insight. It gave me an idea. I can also do a hackaround for now like this:

``docker run --rm -ti --net=host -e KAFKA_TOOLS_LOG4J_LOGLEVEL=DEBUG confluentinc/cp-kafka /bin/bash -c "dub template \"/etc/confluent/docker/tools-log4j.properties.template\" \"/etc/kafka/tools-log4j.properties\"; kafka-console-consumer --bootstrap-server localhost:29092 --topic performanceTest0 --new-consumer --from-beginning"``

It gets the job done for right now, but ideally we'd have this work more cleanly.

**Comment by bencoughlan** [comment]
Is there any update on the 
logging setup for these images? They all have different versions of Log4j installed, which makes for a really inconsistent setup. 

Log4j hasn't been worked on in 6/7 years, is there any reason why they can't all have the newest version? 

Cheers, 
Ben
